rotkehlchen/data_migrations/migrations/migration_6.py

```diff
@@ -87,6 +87,6 @@ def data_migration_6(write_cursor: 'DBCursor', rotki: 'Rotkehlchen') -> None:
     nodes_to_connect = rotki.data.db.get_web3_nodes(blockchain=SupportedBlockchain.ETHEREUM, only_active=True)  # noqa: E501
     # when we sync a remote database the migrations are executed but the chain_manager
     # has not been created yet
-    if hasattr(rotki, 'chain_manger') is True:
-        rotki.chain_manager.ethereum.connect_to_multiple_nodes(nodes_to_connect)
+    if (chain_manager := getattr(rotki, 'chain_manager', None)) is not None:
```

Comment:
No need to change anything now but as we discussed shouldn't setting to None at ctor and then getting rid of this hack be what you do?